portal_frontend/src/pages/access-denied.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+export default function NotFound() {
```

Comment:
This is kind of any anti pattern in my opinion. We should either:
- refresh access tokens
- enlarge access token lifetime
- redirect users to the login page, not to an acces denied page.

Reply:
I'm in for refreshing access tokens